dotnet/src/webdriver/DevTools/DevToolsSessionEventReceivedEventArgs.cs

```diff
@@ -53,6 +54,6 @@ public DevToolsEventReceivedEventArgs(string domainName, string eventName, JsonN
         /// <summary>
         /// Gets the data with which the event is to be raised.
         /// </summary>
-        public JsonNode EventData { get; private set; }
+        public JsonElement EventData { get; private set; }
```

Comment:
This is used all across cdp generated code. Luckily, the generated code looks like `var eventArgs = e.EventData.Deserialize(eventData.EventArgsType);` which is exactly the same when `EventData` is both `JsonNode` or` JsonElement`